src/index.html

```diff
@@ -4,8 +4,8 @@
     <title>Firefox Marketplace</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width">
-
-    <link rel="stylesheet" type="text/css" href="/media/css/bundle.css"/>
+    <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```

Comment:
I'd just download it manually unless they update Fira a lot. The PR for Zamboni should be first anyways since that commit will only affect local development.